Description: Prevent FTBFS for dependent packages.
 Fix small flaw in headers that was causing compilation errors.
 Upstream has been alerted and will take care of this soon.
Author: Jeff Breidenbach <jab@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: vendor
Bug: https://github.com/tesseract-ocr/tesseract/issues/1242
Bug-Debian: http://bugs.debian.org/884903
Forwarded: https://github.com/tesseract-ocr/tesseract/issues/1242
Reviewed-By: jab@debian.org
Last-Update: 2017-12-21

--- tesseract-4.00~git2188-cdc35338.orig/ccutil/unichar.h
+++ tesseract-4.00~git2188-cdc35338/ccutil/unichar.h
@@ -161,7 +161,7 @@ class UNICHAR {
   static std::vector<char32> UTF8ToUTF32(const char* utf8_str);
   // Converts a vector of unicodes to a utf8 string.
   // Returns an empty string if the input contains an invalid unicode.
-  static string UTF32ToUTF8(const std::vector<char32>& str32);
+  static std::string UTF32ToUTF8(const std::vector<char32>& str32);
 
  private:
   // A UTF-8 representation of 1 or more Unicode characters.
